src/main/java/uk/gov/justice/probation/courtcasematcher/messaging/HearingProcessor.java

```diff
@@ -72,6 +73,10 @@ private void matchAndSaveHearing(Hearing receivedHearing, String messageId) {
 
     private void mergeAndUpdateExistingHearing(Hearing receivedHearing, Hearing existingHearing) {
         var courtCaseMerged = HearingMapper.merge(receivedHearing, existingHearing);
+        if(existingHearing.getSource() == LIBRA
```

Comment:
Can we add a unit test for this

Reply:
Not sure if mocking UUID.randomUUID() is feasible with mockito itself... asserting that the hearingId is not equal to existing hearingId is good idea?

Reply:
Yeah I'm fine with that, the key is capturing the two behaviours.